seqr/views/apis/anvil_workspace_api.py

```diff
@@ -174,6 +181,27 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
+    # trigger dags
+    anvil_type = 'AnVIL_{sample_type}'.format(sample_type=request_json['sampleType'])
+
+    anvil_variables = get_variables(anvil_type)
+    updated_anvil_variables = generate_AnVIL_variables(project, data_path, request_json['sampleType'])
+    projects = anvil_variables["active_projects"]
```

Comment:
so for the dag, we no longer care about `active_projects` we care about `projects_to_run`